meson.build

```diff
@@ -0,0 +1,236 @@
+project('pfapack',
+  ['c', 'fortran'],
+  version: '0.3.1',
+  default_options: [
+    'buildtype=release',
+    'c_std=c99',
+    'fortran_std=legacy',
+  ],
+)
+
+# Get compilers
+c_compiler = meson.get_compiler('c')
+fortran_compiler = meson.get_compiler('fortran')
+
+# Dependencies
+thread_dep = dependency('threads')
+m_dep = c_compiler.find_library('m', required: false)  # math library
+
+if host_machine.system() == 'darwin'
+  add_project_link_arguments('-framework', 'Accelerate', language: ['c', 'fortran'])
+  lapack_dep = declare_dependency()
+  blas_dep = declare_dependency()
+elif host_machine.system() == 'windows'
+  # On Windows, find OpenBLAS which includes LAPACK
+  openblas_lib = dependency('openblas', required: false)
+  if not openblas_lib.found()
+    # Fallback to manual detection
+    compiler = meson.get_compiler('c')
+    openblas_lib = compiler.find_library('libopenblas',
+      dirs: ['C:/msys64/mingw64/lib'],
+      required: true)
+  endif
+  # OpenBLAS includes both BLAS and LAPACK
+  lapack_dep = declare_dependency(dependencies: [openblas_lib])
+  blas_dep = declare_dependency(dependencies: [openblas_lib])
+
+else
+  # Add rt library for clock_gettime
+  rt_dep = c_compiler.find_library('rt', required: false)
+  if rt_dep.found()
+    add_project_link_arguments('-lrt', language: ['c', 'fortran'])
+  endif
+
+  # Find LAPACK and BLAS
+  lapack_dep = dependency('lapack', required: true)
+  blas_dep = dependency('blas', required: true)
+endif
+
+pfapack_dir = 'original_source'
+
+# Fortran sources in correct order matching the Makefile.FORTRAN
+fortran_sources = files(
+  # Single precision real
+  join_paths(pfapack_dir, 'fortran', 'slasktrd.f'),
```

Comment:
Thanks for the recommendation!

The only reason I had this was because I was still not sure whether I wanted to call the directory  `original_source` or something else. So I thought if I would change it the diff would be smaller 🙈